examples/hello-world/hello-flower/job.py

```diff
@@ -12,45 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
 from argparse import ArgumentParser
 
-from nvflare.app_opt.flower.flower_pt_job import FlowerPyTorchJob
-from nvflare.client.api import ClientAPIType
-from nvflare.client.api_spec import CLIENT_API_TYPE_KEY
+from nvflare.app_opt.flower.recipe import FlowerRecipe
+from nvflare.recipe import SimEnv, add_experiment_tracking
 
 
 def main():
     parser = ArgumentParser()
     parser.add_argument("--job_name", type=str, required=True)
     parser.add_argument("--content_dir", type=str, required=True)
     parser.add_argument("--stream_metrics", action="store_true")
-    parser.add_argument("--use_client_api", action="store_true")
```

Comment:
not just streaming, if they want to use ANY client api functionality we need to set that env value.

Reply:
i see you already set it, ok